src/app/containers/ATIAnalytics/canonical/index.test.jsx

```diff
@@ -20,27 +19,17 @@ describe('Canonical ATI Analytics', () => {
     beacon.default = mockSendBeacon;
 
     act(() => {
-      create(<CanonicalATIAnalytics pageviewParams={mockPageviewParams} />);
+      render(<CanonicalATIAnalytics pageviewParams={mockPageviewParams} />);
     });
 
     expect(mockSendBeacon).toHaveBeenCalledTimes(1);
     expect(mockSendBeacon).toHaveBeenCalledWith(expectedUrl);
   });
 
   it('should render a noscript image for non-JS users', () => {
-    const renderedATI = render(
+    const { container } = render(
       <CanonicalATIAnalytics pageviewParams={mockPageviewParams} />,
     );
-    expect(renderedATI).toMatchSnapshot();
```

Comment:
Do we lose any value binning the snapshot? Snapshots might need updating to reflect this disappearing too 👌

Reply:
with react testing library (and in the browser) you cant view the contents of the noscript element with a DOM query so it's just a snapshot of an empty noscript tag

Reply:
I think an obsolete snapshot does need removing now that this is in the bin